src/output.cpp

```diff
@@ -1600,6 +1619,40 @@ void get_HP_Bar(const int current_hp, const int max_hp, nc_color &color, std::st
     }
 }
 
+void get_item_HP_Bar(const int iDamage, nc_color &color, std::string &text)
```

Comment:
Why not return a `std::pair` here? It would also shorten the code switch: `return std::make_pair(...)`.